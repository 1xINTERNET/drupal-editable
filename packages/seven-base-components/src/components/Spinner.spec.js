/* eslint-env jest */

import React from "react";
import { shallow } from "enzyme";

import { Spinner } from ".";

describe("Spinner", () => {
  let component;

  beforeEach(() => {
    component = shallow(<Spinner />);
  });

  it("should render correctly", () => {
    expect(component).toMatchSnapshot();
  });
});
