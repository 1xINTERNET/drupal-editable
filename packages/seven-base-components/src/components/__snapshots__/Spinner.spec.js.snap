// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render correctly 1`] = `
<div
  aria-label="Loading"
  className="css-h0lbk4 editable--spinner"
/>
`;
